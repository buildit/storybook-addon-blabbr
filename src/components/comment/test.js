import React from 'react';
import renderer from 'react-test-renderer'; // eslint-disable-line
import Comment from './';

test('Comment renders correctly', () => {
  const tree = renderer.create(
    <Comment
      emailId="email@id.com"
      username="username"
      timestamp="1-Jan-17"
      comment="This is a comment"
      commentId="id123"
      currentUserIsOwner
      handleEditUserComment={() => {}}
      handleDeleteUserComment={() => {}}
      edited
      lastUpdated="2-Jan-17"
    />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
