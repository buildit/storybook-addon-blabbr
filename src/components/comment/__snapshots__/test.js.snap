// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment renders correctly 1`] = `
<article
  className="blabbr-comment withAvatar"
>
  <header>
    <h2>
      username
    </h2>
    <span
      className="blabbr-time"
    >
      at 
      <time
        dateTime="1-Jan-17"
      >
        1-Jan-17
      </time>
    </span>
    <img
      alt="username's Gravatar"
      className="avatar"
      src="https://gravatar.com/avatar/7b707d018e06cb8c85737c876034a655?s=40&r=pg&d=retro"
    />
    <span
      className="controls"
    >
      <button
        id="id123"
        onClick={[Function]}
      >
        Edit
      </button>
      <button
        className="remove"
        id="id123"
        onClick={[Function]}
      >
        Remove
      </button>
    </span>
  </header>
  <p>
    This is a comment
  </p>
  <p>
    <small>
      (edited - 
      2-Jan-17
      )
    </small>
  </p>
</article>
`;
